pom.xml

```diff
@@ -46,7 +46,11 @@
 			<groupId>org.springframework.boot</groupId>
 			<artifactId>spring-boot-starter-validation</artifactId>
 		</dependency>
-
+		<dependency>
+			<groupId>org.springframework.security</groupId>
+			<artifactId>spring-security-crypto</artifactId>
+			<version>6.1.0</version>
+		</dependency>
```

Comment:
I used that dependency because of ticket scope. If i used spring boot security starter dependency i would have to disable it just so i can test the encryption part

Reply:
> I used that dependency because of ticket scope. If i used spring boot security starter dependency i would have to disable it just so i can test the encryption part

Why would you need to disable it?

Reply:
Because it is required to login with predefined credentials. I tried with that and even with predefined credentials i cannot login.